src/Model/IssueModel.php

```diff
@@ -41,4 +49,10 @@ public function getBody(): string
     {
         return $this->body;
     }
+
+    public function getCreatedAt(): DateTime
+    {
+        $this->createdAt = new DateTime();
```

Comment:
This is not needed as mentioned above. The JMS\Serializer already transforms the incoming date string from `created_at` into the proper type.